sourcecode/apis/contentauthor/tests/Integration/Libraries/QuestionSetConverterTest.php

```diff
@@ -89,19 +86,19 @@ public function testCreateH5PQuestionSet(): void
 
         /** @var QuestionSetConvert $questionsetConverter */
         $questionsetConverter = app(QuestionSetConvert::class);
-        list($id, $title, $machineName, $route, $resourceType) = $questionsetConverter->convert(
+        $h5p = $questionsetConverter->convert(
             H5PQuestionSet::$machineName,
             $questionSet,
             $resourceMetaObject
         );
 
+        $this->assertInstanceOf(H5PContent::class, $h5p);
         $this->assertDatabaseHas('h5p_contents', [
-            'id' => $id,
-            'title' => $title,
+            'id' => $h5p->id,
+            'title' => $questionSet->title,
             'license' => License::LICENSE_BY_NC,
         ]);
-        $this->assertEquals(H5PQuestionSet::$machineName, $machineName);
-        $this->assertEquals(route('h5p.edit', $id), $route);
-        $this->assertEquals(Content::TYPE_H5P, $resourceType);
+        // FIXME: when is this stored?
```

Comment:
The content is stored to database in `app/libraries/H5P/Framework.php::insertContent()`, the library is set in `app/Libraries/H5P/h5p.php::storeContent()`
The library the QS is converted to, specified in `app/Libraries/H5P/Packages/QuestionSet.php`, does not exist in the DB, and the h5p will get a `library_id` of `0`